### PR TITLE
https://issues.redhat.com/browse/ACM-5380 PlacementRule deprecation

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-5380

### CODE REVIEW

File: governance/create_policy.adoc [hunk 1]
```diff
@@ -24,11 +24,11 @@ View the following sections:
 You can create a security policy from the command line interface (CLI) or from the console.
 
 **Required access**: Cluster administrator
-
+//PlacementRule here
 *Important:* You must define a placement rule and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementRule` and `PlacementBinding`.
```
Comment (dockerymick) [inline governance/create_policy.adoc]:
```suggestion
*Important:* You must define a placement rule (deprecated) or placement, and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementRule` (deprecated) or `Placement` and `PlacementBinding`.
```

**Comment by ch-stark** [comment]
@dockerymick will we also add a deprecation note?

File: governance/create_policy.adoc [hunk 2]
```diff
@@ -24,11 +24,11 @@ View the following sections:
...
 **Required access**: Cluster administrator
-
+//PlacementRule here
 *Important:* You must define a placement rule and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementRule` and `PlacementBinding`. 
 
 See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements[Resources that support support set-based requirements] in the Kubernetes documentation for a valid expression. View the definitions of the objects that are required for your {product-title-short} policy:
-
+//PlacementRule here
 * _PlacementRule_: Defines a _cluster selector_ where the policy must be deployed.
 * _PlacementBinding_: Binds the placement to a placement rule.
```
Comment (dockerymick) [inline governance/create_policy.adoc]: Does Placement have a different definition from Placement rule? Currently the definition used in the RHACM [Glossary](https://github.com/stolostron/rhacm-docs/blob/2.8_stage/about/glossary_terms.adoc#placement-policy) is, "A policy that defines where the application components are deployed and how many replicas there are." 

Is it accurate for us to add that definition to this list?

File: governance/create_policy.adoc [hunk 3]
```diff
@@ -24,11 +24,11 @@ View the following sections:
...
 
 **Required access**: Cluster administrator
-
+//PlacementRule here
 *Important:* You must define a placement rule and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementRule` and `PlacementBinding`. 
 
 See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements[Resources that support support set-based requirements] in the Kubernetes documentation for a valid expression. View the definitions of the objects that are required for your {product-title-short} policy:
-
+//PlacementRule here
 * _PlacementRule_: Defines a _cluster selector_ where the policy must be deployed.
```
Comment (dockerymick) [inline governance/create_policy.adoc]:
```suggestion
* _PlacementRule_ (deprecated): Defines a _cluster selector_ where the policy must be deployed.
```

File: governance/deploy_gitops.adoc [hunk 4]
```diff
@@ -158,7 +158,7 @@ After you run the `deploy.sh` script, any user with access to the repository can
 
 . Bind the `open-cluster-management:subscription-admin` ClusterRole to the user creating the subscription.
 . If you are using an allow list in the subscription, include the following API entries:
-
+//we can possibly add a comment to say PlacementRule is deprecated and the add Placement before PlacementRule
```
Comment (dockerymick) [inline governance/deploy_gitops.adoc]:
```suggestion
//we can possibly add a comment to say PlacementRule is deprecated and the add Placement before PlacementRule (the info that is listed in the YAML example. PlacementRule is listed as a kind after `apiVersion: apps.open-cluster-management.io/v1`
```

File: governance/policy_ctrl_adv_config.adoc [hunk 5]
```diff
@@ -85,7 +85,7 @@ The policy framework exposes metrics that show policy distribution and complianc
 The `policy_governance_info` is collected by the {ocp-short} monitoring feature, and some aggregate data is collected by {product-title-short} observability, if it is enabled.
 
 *Note:* If observability is enabled, you can enter a query for the metric from the Grafana _Explore_ page. 
-
+//PlacementRule used here
```
Comment (dockerymick) [inline governance/policy_ctrl_adv_config.adoc]:
```suggestion
//PlacementRule used here, line 89
```

File: release_notes/deprecate_remove.adoc [hunk 6]
```diff
@@ -167,6 +167,12 @@ A _deprecated_ component, feature, or service is supported, but no longer recomm
 |===
 | Product or category | Affected item | Version | Recommended action | More details and links
 
+| Governance
+| `PlacementRule` resource
+| 2.8
+| Use the `Placement` resource.
+| See <Need to add a link if applicable>
```
Comment (dockerymick) [inline release_notes/deprecate_remove.adoc]: @ch-stark the deprecation note is here

File: governance/create_policy.adoc [hunk 7]
```diff
@@ -84,7 +84,7 @@ spec:
                   verbs: ["get", "list", "watch", "create", "delete","patch"]
 ----
 
-. Define a `PlacementRule`. Be sure to change the `PlacementRule` to specify the clusters where the policies need to be applied by adjusting the `clusterSelector`. View link:../applications/placement_sample.adoc#placement-rule-samples[Placement rule samples overview]
+. *Deprecated* Define a `PlacementRule`. Be sure to change the `PlacementRule` to specify the clusters where the policies need to be applied by adjusting the `clusterSelector`. View link:../applications/placement_sample.adoc#placement-rule-samples[Placement rule samples overview]
```
Comment (dockerymick) [inline governance/create_policy.adoc:87]:
```suggestion
. *Deprecated* Define a `PlacementRule`. Be sure to change the `PlacementRule` to specify the clusters where the policies need to be applied by adjusting the `clusterSelector`. View link:../applications/placement_sample.adoc#placement-rule-samples[Placement rule samples overview]
+
*Note:* Use `Placement` instead.
```

File: governance/policy_ctrl_adv_config.adoc [hunk 8]
```diff
@@ -85,7 +85,7 @@ The policy framework exposes metrics that show policy distribution and complianc
 The `policy_governance_info` is collected by the {ocp-short} monitoring feature, and some aggregate data is collected by {product-title-short} observability, if it is enabled.
 
 *Note:* If observability is enabled, you can enter a query for the metric from the Grafana _Explore_ page. 
-
+//PlacementRule used here, line 89
```
Comment (dockerymick) [inline governance/policy_ctrl_adv_config.adoc]:
```suggestion
```

File: governance/policy_ctrl_adv_config.adoc [hunk 9]
```diff
@@ -85,7 +85,6 @@ The policy framework exposes metrics that show policy distribution and complianc
 The `policy_governance_info` is collected by the {ocp-short} monitoring feature, and some aggregate data is collected by {product-title-short} observability, if it is enabled.
 
 *Note:* If observability is enabled, you can enter a query for the metric from the Grafana _Explore_ page. 
-
 When you create a policy, you are creating a _root_ policy. The framework watches for root policies, as well as `PlacementRules` and `PlacementBindings`, to determine where to create _propagated_ policies in order to distribute the policy to managed clusters.
```
Comment (dockerymick) [inline governance/policy_ctrl_adv_config.adoc]:
```suggestion
When you create a policy, you are creating a _root_ policy. The framework watches for root policies, as well as `PlacementRules` (deprecated) or `Placement`, and `PlacementBindings` to determine where to create _propagated_ policies in order to distribute the policy to managed clusters.
```

File: governance/policy_overview.adoc [hunk 10]
```diff
@@ -8,7 +8,7 @@ Each {product-title-short} policy can have at least one or more templates. For m
 The policy requires a _PlacementRule_ or _Placement_ that defines the clusters that the policy document is applied to, and a _PlacementBinding_ that binds the {product-title} policy to the placement rule. For more on how to define a `PlacementRule`, see link:../applications/app_model.adoc#placement-rules[Placement rules] in the Application lifecycle documentation. For more on how to define a `Placement` see link:../clusters/cluster_lifecycle/placement_managed.adoc#placement-overview[Placement overview] in the Cluster lifecycle documentation.
 
 *Important:*
-
+//PlacementRule here
 * You must create the `PlacementBinding` to bind your policy with either a `PlacementRule` or a `Placement` in order to propagate the policy to the managed clusters.
```
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
* You must create the `PlacementBinding` to bind your policy with either a `PlacementRule` (deprecated) or a `Placement` in order to propagate the policy to the managed clusters.
```

File: release_notes/deprecate_remove.adoc [hunk 11]
```diff
@@ -168,7 +168,7 @@ A _deprecated_ component, feature, or service is supported, but no longer recomm
 |===
 | Product or category | Affected item | Version | Recommended action | More details and links
 
-| Applications
+| Applications and Governance
 | `PlacementRule`
 | 2.8
 | Use `Placement` anywhere that you might use `PlacementRule`
```
Comment (dockerymick) [inline release_notes/deprecate_remove.adoc]:
```suggestion
| Use `Placement` anywhere that you might use `PlacementRule`.
```

File: governance/create_policy.adoc [hunk 12]
```diff
@@ -25,7 +25,7 @@ You can create a security policy from the command line interface (CLI) or from t
 
 **Required access**: Cluster administrator
 
-*Important:* You must define a placement rule and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementRule` and `PlacementBinding`. 
+*Important:* You must define a placement rule (deprecated) or placement, and placement binding to apply your policy to a specific cluster. Enter a valid value for the _Cluster selector_ field to define a `PlacementRule` (deprecated) or `Placement` and `PlacementBinding`.
```
Comment (cadawson) [inline governance/create_policy.adoc:28]: Should this be `PlacementRule`, or does it matter here?

**Review by cadawson — APPROVED** [review]
Looks good. Question added, but minor. Thanks!

File: governance/create_policy.adoc (hunk 12 above)
Comment (dockerymick) [reply governance/create_policy.adoc:28]: I use the `kind` value bc that is what the user would see in the YAML editor, from the console

**Comment by dockerymick** [comment]
@dhaiducek thanks for doing the live review with me. When you have a chance, will you approve the PR if you agree with the changes

**Review by dhaiducek — APPROVED** [review]
LGTM! Thanks, @dockerymick!

**Comment by dockerymick** [comment]
Will handle the anchor update in a different PR

**Pull request merged**